Refactor to chart (#30)
alan-turing-institute/hub23-deploy: README.md

`generate-configs.py` will populate `secret-template.yaml` and `config-template.yaml` with the appropriate information and save the output as `.secret/secret.yaml` and `.secret/config.yaml`.

> `.secret/` is a git-ignored folder so that the `secret.yaml` and `prod.yaml` files (and any secrets downloaded in the process of creating them) cannot be pushed to GitHub.

Author: ```suggestion
`.secret/` is a git-ignored folder so that the secrets contained in `.secret/prod.yaml` cannot be pushed to GitHub.
```
